saleor/page/migrations/0008_raw_html_to_json.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 2.2.3 on 2019-07-15 07:51
+
+from django.db import migrations
+from html_to_draftjs import html_to_draftjs
+
+
+def convert_pages_html_to_json(apps, schema_editor):
+    Page = apps.get_model("page", "Page")
+    qs = Page.objects.filter(content_json={})
+
+    for page in qs:
+        page.content_json = html_to_draftjs(page.content)
```

Comment:
Maybe this should happen only for fields that have empty `_json` data? The thing is that even if don't have any content, DraftJS could've already created an empty structure like:
```
{'blocks': [{'key': 'as2m7',
   'data': {},
   'text': 'Test',
   'type': 'unstyled',
   'depth': 0,
   'entityRanges': [],
   'inlineStyleRanges': []}],
 'entityMap': {}}
```
So simple comparison with `None` or empty string wouldn't work.

Reply:
You mean seeing if the plain text is empty then?